python/rmm/tests/test_rmm.py

```diff
@@ -942,3 +942,33 @@ def test_rmm_device_buffer_copy(cuda_ary, make_copy):
     result = db_copy.copy_to_host()
 
     np.testing.assert_equal(expected, result)
+
+
+@pytest.mark.parametrize("level", rmm.logging_levels)
+def test_valid_logging_level(level):
```

Comment:
What happens if the Python module was compiled with a higher logging level like `err`? Setting the level to `trace` will not show `trace` messages, right? Is there a way to test this programmatically, or show warnings when setting a low level like `trace`, so that users can be aware if their chosen logging level does nothing because of the Python module's compiled setting?

Reply:
A warning is a good idea.

Reply:
Warning added.